ansible/roles/stack-sunbird/templates/sunbird_lms-service.env

```diff
@@ -104,4 +104,6 @@ sunbird_user_org_api_base_url={{sunbird_user_org_api_base_url}}
 kafka_topics_instruction={{kafka_topics_instruction}}
 kafka_urls={{kafka_urls}}
 kafka_topics_certificate_instruction={{kafka_topics_certificate_instruction}}
-kafka_assessment_topic={{kafka_assessment_topic}}
\ No newline at end of file
+kafka_assessment_topic={{kafka_assessment_topic}}
+# Required by lms service to call cert service
+sunbird_cert_service_base_url=http://cert-service:9000
```

Comment:
change this variable to 
` sunbird_cert_service_base_url={{sunbird_cert_service_base_url | d('http://cert-service:9000')}}`